templates/base.html

```diff
@@ -35,7 +35,8 @@
                             <li style="float: center;"><a class="nav-color" href="{% url 'profile' %}">PROFILE</a></li>
                             <li style="float: center;"><a class="nav-color" href="{% url 'logout' %}">LOGOUT</a></li>
                         {% else %}
-                            <li style="float: center;"><a data-open="login-modal">LOGIN</a></li>
+                            <li style="float: center;"><a class="nav-color" href="{% url 'register' %}">SIGN UP</a></li>
```

Comment:
With a "Sign up" button directly in the nav, do you think we should remove the "...or register an account" from the Login pop-up modal?